spec/services/activities/import_from_csv_spec.rb

```diff
@@ -259,6 +262,7 @@
       expect(new_activity.sector).to eq(new_activity_attributes["Sector"])
       expect(new_activity.sector_category).to eq("112")
       expect(new_activity.collaboration_type).to eq(new_activity_attributes["Collaboration type (Bi/Multi Marker)"])
+      expect(new_activity.flow).to eq(existing_activity_attributes["Flow"])
```

Comment:
I see that you've fixed this to use `new_activity` in the next commit, but in the interest of green commits and whatnot, would you move the fix to this commit? :)

Reply:
Done!